src/models/classifiers.py

```diff
@@ -50,26 +51,14 @@ def metrics(classifier_name, y_train_predict, y_test_predict, y_train, y_test):
     with open('models/metrics.json', 'w') as file:
         json.dump(data, file, indent=2)
         
-        
-clf_sgd = SGDClassifier()
-clf_sgd.fit(X_train, y_train)
+rf_clf = RandomForestClassifier()
```

Comment:
Давай обернем это в функцию.
```suggestion
def train(X_train, y_train):
    rf_clf = RandomForestClassifier()
    f_clf.fit(X_train, y_train)
    и так далее
```